mapping-for-navigation/_posts/0000-01-04-adding-turn-restrictions.md

```diff
@@ -239,7 +245,7 @@ _An example of a `restriction=no_right_turn` traffic sign in Germany_
 ![screen shot 2016-08-17 at 4 27 33 pm](https://cloud.githubusercontent.com/assets/11845908/17734070/a6125f5e-6497-11e6-8d70-d55248e938ec.png)
 
 *Courtesy: [German wiki](http://wiki.openstreetmap.org/wiki/DE:Relation:restriction)*
- 
+
 
```

Comment:
Can we also add this example as a part of QA @krishnanammala https://github.com/mapbox/navigation-data/issues/115#issuecomment-364248697

Reply:
@jothirnadh Thanks for reviewing , FYI I have only edited the iD version of the guide not the complete one , because that is part of restructuring. Related PR https://github.com/mapbox/mapping/pull/346 which is not a priority for now. 

But I agree regarding adding the edge cases to this guide with the examples we have so far. Adding those is also under low priority Related PR https://github.com/mapbox/mapping/pull/329

Reply:
👍